### DISCUSSION TITLE
Brian Fire discussion

### DISCUSSION THREAD

**Discussion by gcarreno** [ObjectPascal-Community/1brc-ObjectPascal#59] [General]
Hey Brian(@EagleAglow),

Almost there :rofl: :
```console
$ ./bin/bfire -i /tmp/measurements-1_000_000_000.txt 
ERROR: Missing input file flag.
```

Cheers,
Gus

**Comment by gcarreno** [comment]
Hey Brian(@EagleAglow),

Completely forgot I could, temporarily, rename the file... DOH!!! :facepalm: 

On a semi-busy system and a cold run:
```console
$ time ./bin/bfire -i /tmp/measurements_1_000_000_000.txt
{...}
real	10m4.383s
user	9m53.744s
sys	0m10.016s
```

**Comment by gcarreno** [comment]
Hey Brian(@EagleAglow),

Houston we have a problem !!
You cannot use `WinApi.Windows` on a Linux64 Target.
Will have to skip running your entry if you can't correct that ASAP!!

Cheers,
Gus

**Reply by EagleAglow** [reply]
Sorry, I thought I had removed that unit already.  I call "Sleep", but it should work in LINUX without the Windows unit.
